projects/ccf-shared/src/lib/components/organ-selector/organ-selector.component.html

```diff
@@ -1,11 +1,14 @@
-<div class="container" [class.error]="error" matTooltip="Select the organ your tissue block is from.">
+<div class="container" [class.error]="error" [matTooltipDisabled]="multiselect" matTooltip="Select the organ your tissue block is from.">
   <mat-icon class="carousel-left-icon" (touchstart)="scroll('left')" (touchend)="stopScroll()" (click)="shift('left')" (mousedown)="scroll('left')" (mouseup)="stopScroll()" [class.disabled]="onLeft">chevron_left</mat-icon>
   <div class="carousel-item-container" #itemcontainer>
-    <div class="carousel-item-list" [class.hidden]="hidden" #itemlist>
+    <div class="carousel-item-list" #itemlist>
       <div *ngFor="let organ of organList" class="carousel-item" [class.disabled]="organ.disabled" (click)="selectOrgan(organ)"
           [class.selected]="isSelected(organ)">
           <mat-icon class="carousel-icon" svgIcon="{{organ.src}}"></mat-icon>
-          <span class="carousel-text">{{organ.name}}</span>
+          <div class="carousel-text-container">
+            <div class="carousel-text">{{ organ.name }}</div>
+            <div class="num-results" *ngIf="organ.numResults !== undefined">{{ organ.numResults ? organ.numResults : 0 }}</div>
```

Comment:
You shouldn't need the extract tertiary logic now that we're hiding when numResults is undefined.